### PR TITLE
[Macquarie.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by youdly** [comment]
Travis: `Unable to find image 'node:latest' locally`.

**Comment by youdly** [comment]
Rebased to fix travis error

**Comment by youdly** [comment]
Rebased. Still `Unable to find image 'node:latest' locally`

**Comment by youdly** [comment]
Moved `Macquarie.com.au.xml` to separate PR https://github.com/EFForg/https-everywhere/pull/13169

File: src/chrome/content/rules/Macquarie.com.xml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+	<target host="pb-lab.macquarie.com" />
+	<target host="pb-opdcs8.macquarie.com" />
+	<target host="staffid.macquarie.com" />
+	<target host="static.macquarie.com" />
+	<target host="static-origin.macquarie.com" />
+
+	<target host="www.macquarieconnections.com" />
+	<target host="www.macquarieresearch.com" />
+
+	<securecookie host="^www\.macquarie\.com$" name=".*" />
```
Comment (J0WI) [inline src/chrome/content/rules/Macquarie.com.xml]: Should be `.+`

**Pull request merged**